cnd/src/config/mod.rs

```diff
@@ -28,11 +30,37 @@ pub struct Socket {
 pub struct Bitcoin {
     #[serde(with = "crate::config::serde_bitcoin_network")]
     pub network: bitcoin::Network,
+    pub bitcoind: Bitcoind,
+}
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct Bitcoind {
     pub node_url: reqwest::Url,
 }
 
+impl Default for Bitcoin {
+    fn default() -> Self {
+        Self {
+            network: bitcoin::Network::Regtest,
+            bitcoind: Bitcoind::defualt(),
+        }
+    }
+}
+
+impl Default for Bitcoind {
+    fn default() -> Self {
+        Self {
+            node_url: "http://localhost:18443",
```

Comment:
Shouldn't this depend on the network?

I think you probably can't use the `Default` trait here. If the user says `Network::Mainnet` but leaves this out, it should default to `http://localhost:8443`.

Reply:
Good point. I'll see how I can resolve this.